tests/conftest.py

```diff
@@ -1,8 +1,39 @@
+import os
 from pathlib import Path
 
 import pytest
+from matplotlib import pyplot as plt
+from matplotlib.figure import Figure
+from matplotlib.testing.compare import compare_images
 
 
 @pytest.fixture
 def root_dir():
     return Path(__file__).resolve().parent
+
+
+# simplified from https://github.com/scverse/scanpy/blob/main/scanpy/tests/conftest.py
+@pytest.fixture
+def check_same_image(tmp_path):
```

Comment:
Oh yeah, time to get started with this :) I've been very lazy about it.
Should also be another issue.........

But hey - as soon as the revisions are over we can tackle this stuff as well

Reply:
Opened issue #666 to that 👍